src/CascadeSoftDeletes.php

```diff
@@ -117,7 +117,7 @@ protected function getCascadingDeletes()
     protected function getActiveCascadingDeletes()
     {
         return array_filter($this->getCascadingDeletes(), function ($relationship) {
-            return ! is_null($this->{$relationship});
+            return ! is_null($this->{$relationship}());
```

Comment:
If this is going to operate on the relationship directly maybe it should use the Elloquent `exists()` method instead.

```suggestion
            return $this->{$relationship}()->exists();
```

Reply:
Of course. Thanks. I updated the PR.